packages/components/src/components/skeleton/skeleton.tsx

```diff
@@ -0,0 +1,54 @@
+import clsx from 'clsx';
+import React from 'react';
+
+type TSkeletonProps = {
+    animated?: boolean;
+    className?: string;
+    rows?: number;
+    variant?: typeof VARIANT[keyof typeof VARIANT];
+} & React.CSSProperties;
+
+export const VARIANT = {
+    ICON: 'icon',
+    PARAGRAPH: 'paragraph',
+} as const;
+
+const Skeleton = ({
+    animated = true,
+    className,
+    gap,
+    height,
+    rows,
+    width,
+    variant,
+    ...css_properties
+}: TSkeletonProps) => {
+    if (VARIANT.PARAGRAPH && rows) {
+        return (
+            <div className='skeleton-paragraph' data-testid='dt_skeleton_paragraph' style={{ gap: gap ?? 8 }}>
```

Comment:
will it work without `px`? I mean just 8

Reply:
yes, it does work perfectly)